### DISCUSSION TITLE
Ability to point TVA at a Signal?

### DISCUSSION THREAD

**Discussion by fsqonbdk** [akumidv/tradingview-assistant-chrome-extension#126] [Ideas]
Hi, love using TVA to explore Strategies, but I also use a Signal piped into a Strategy (for example, see Template Trailing Strategy which accepts inputs from Signals). Is it possible for TVA to work with a signal, or if not, could this be put in as a feature request?

Thank you!

**Comment by akumidv** [comment]
It's impossible, due all calculation are making on tradingview backend side and signals it's just showed on charts.
